experimental/Schemes/SpaceGerms.jl

```diff
@@ -63,37 +201,120 @@ end
     return Spec(R)
 end
 
-function point(X::SpaceGerm{<:Any, <:Any, <:GermAtClosedPoint})
+@doc raw"""
+    point(X::AbsSpaceGerm)
+
+Returns the point `p` of a germ `(X,p)`.
+
+# Examples:
+```jldoctest
+julia> R, (x,y,z) = QQ["x", "y", "z"];
+
+julia> I = ideal(R, [(x-1)*(x^2 - y^2 + z^2)]);
+
+julia> X = Spec(R, I);
+
+julia> XS = SpaceGerm(X,[0,0,0]);
+
+julia> point(XS)
+3-element Vector{QQFieldElem}:
+ 0
+ 0
+ 0
+
+```
+"""
+function point(X::AnySpaceGermClosedPoint)
   return point_coordinates(inverted_set(OO(X)))
 end
 
 ## currently unused case
-function point(X::SpaceGerm{<:Any, <:Any, <:GermAtGeometricPoint})
+function point(X::AnySpaceGermGeometricPoint)
   return prime_ideal(inverted_set(OO(X)))
 end
 
 Oscar.ring(X::AbsSpaceGerm) = OO(X)
```

Comment:
This should probably be declared on a higher level if it's not there, yet. Otherwise, it's superfluous. Let's check which one is the case.

Reply:
But for the moment, I would prefer to leave it there and just put a TODO.